README.md

```diff
@@ -5,6 +5,7 @@ A public collection of useful scripts to help manage the Mellanox BlueField
 SoC.
 
 Overview of each file:
+- **bfauxpwr** Config ACPI daemon to handle AUX power mode event.
```

Comment:
Suggest make this script more useful by adding command arguments and functions, such as:

Functions to disable and enable all the cores other than 0;
Function to apply ACPI config (the changes you are doing now);
In future all related core and power management could be added in this script.